### PR TITLE
App Crash on missing string element

### PR DESCRIPTION
**Pull request by n00blet**
I have added the dark sky api key, to prevent app crash.

### CODE REVIEW

**Comment by alexbate** [comment]
You almost certainly don't want to put your own API key here, unless you're happy to pay for everyone using this project.

**Comment by n00blet** [comment]
Only 1000 requests/day, and then it will only be charged if I am willing to pay, else those extra requests will be discarded :) So, everyone doesn't have to create api keys to test the app

**Comment by akodiakson** [comment]
I think it would be kind to be unobtrusive to the `build.gradle`, `gitignore` and the `Activity` implementation, changing only the elements that actually fix the issue. The issue seems intentional as @n00blet and the author mentioned.

File: app/build.gradle [hunk 1]
```diff
@@ -1,13 +1,13 @@
 apply plugin: 'com.android.application'
```
Comment (t8erjoe) [inline app/build.gradle]: plugin: 'com.android.applicat

**Pull request closed without merging**